tests/test_events.py

```diff
@@ -28,41 +28,49 @@ async def _ignore_test_consoleLog_logEntryAddedEventEmitted(websocket,
             "expression": "console.log('some log message')",
             "target": {"context": context_id}}})
 
+    # The order of the two responses are not guarenteed
+    log_entry_added_event_response = await read_JSON_message(websocket)
+    command_done_response = await read_JSON_message(websocket)
+    if "id" in log_entry_added_event_response:
+        log_entry_added_event_response, command_done_response = \
+            command_done_response, log_entry_added_event_response
```

Comment:
Oh, you can add a helper function `wait_for_event` in the [helper](https://github.com/GoogleChromeLabs/chromium-bidi/blob/main/tests/_helpers.py) to avoid this trick!